sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/TableUtils.java

```diff
@@ -97,6 +97,26 @@ public static Throwable mapThrowableToTableServiceException(Throwable throwable)
         }
     }
 
+    /**
+     * Map a {@link Throwable} to {@link IllegalArgumentException} if the endpoint
+     * the table name, else it returns the original throwable
+     * @param throwable A throwable exception
```

Comment:
nit; Technically a `Throwable` can be something other than an `Exception`, like an `Error`, for example.

Reply:
Maybe we could say this?
```suggestion
     * @param throwable The throwable to map.
```

Reply:
Sure